docs/docs/config.md

```diff
@@ -36,6 +36,7 @@ replication_password           | STRING               | password on replication
 replication_port               | INT                  | port on replication server                          | 3306
 replication_user               | STRING               | user on replication server                          |
 replication_ssl                | [SSL_OPT](#sslopt)   | SSL behavior for replication cx cx                  | DISABLED
+replication_heartbeat          | BOOLEAN              | enable replication heartbeats to detect stale connections | ENABLED
```

Comment:
let's leave this disabled by default for the world until Zendesk runs the code through its paces with it off.  Happy to enable it by default in a future release once it has proved itself.